### PR TITLE
Javaprograms

### PR DESCRIPTION
**Pull request by vamsi451**

### CODE REVIEW

File: src/com/vamshi/javaprogramplementation.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.vamshi;
+
+public class javaprogramplementation extends Javaprog{
```
Comment (merugaJS) [inline src/com/vamshi/javaprogramplementation.java]: extending is not needed

File: src/com/vamshi/javaprogramplementation.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+	public static void main(String[] args) {
+		javaprogramplementation j = new javaprogramplementation();
```
Comment (merugaJS) [inline src/com/vamshi/javaprogramplementation.java]: create object of class Javaprog
`Javaprog program = new Javaprog()`

File: src/com/vamshi/javaprogramplementation.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+		System.out.println(j.averagegrade());
+        System.out.println("CheckLeapYear"+' '+j.CheckLeapYear());
+        System.out.println(j.factorial());
```
Comment (merugaJS) [inline src/com/vamshi/javaprogramplementation.java:9]: once you have object for Javaprog
you can call methods for those classes using that object

**Review by merugaJS — CHANGES_REQUESTED** [review]

**Comment by vamsi451** [comment]
@merugaJS changes are made!

**Review by merugaJS — APPROVED** [review]

**Pull request merged**